sequencer/src/bin/deploy.rs

```diff
@@ -122,6 +126,18 @@ struct Options {
     #[clap(short, long)]
     pub use_mock: bool,
 
+    /// Option to deploy contracts owned by multisig
+    #[clap(long, default_value = "false")]
+    pub use_multisig: bool,
+
+    /// Option to test upgrade stake table v2 multisig owner dry run
+    #[clap(long, default_value = "false")]
+    pub dry_run: bool,
+
+    /// Option to test locally but with a real eth network
```

Comment:
What does "real" mean here?

Reply:
called it mock-espresso-live-network as it does use a live eth network so didn't want it to mask that fact

https://github.com/EspressoSystems/espresso-network/pull/3155/commits/335847c2edc0fb71476877ba0986ed543d477e3f